2018q4/PAE.md

```diff
@@ -0,0 +1,34 @@
+## i386 PAE Pagetables ##
+
+Contact: Konstantin Belousov, <kib@freebsd.org>
+
+Link:	 https://reviews.freebsd.org/D18894
+
+The i386 architecture (in modern terms, x86 architecture in 32bit
+protected mode), has supported hardware execute-disable since early
+200x.  The only problem preventing the i386 FreeBSD kernel from using
+it was that default page table format used by the kernel is 2-level
+paging, while nx bit is only available for PAE (2.5 levels) page table
+structures.  PAE option is too intrusive, it changes both vm_paddr_t
+and bus_addr_t to 64bit, which is not too friendly to many drivers.
+
+I tried to provide PAE_PAGETABLES kernel option which only changed
+page table format, without affecting vm_paddr_t or bus_addr_t, thus
+keeping kernel/driver interfaces intact.  But I was not able to make
+i386 releases carry two kernels, one to support relic hardware which
+cannot use PAE pagetables, and another for newer machines.
+
+So I finally did a merge which makes single i386 kernel carry two pmap
+modules, one for PAE and one for old two-level paging structures.
+Also I did not found a reason to not expand vm_paddr_t, while I have
```

Comment:
This sentence is a bit awkward as it contains both a double negative as well as an irregular verb. Did you mean "Also I found a reason not to expand vm_paddr_t" or "Also I found a reason to expand vm_paddr_t"?

Reply:
Alright, since that is how you want it, I shan't argue it (since a double negative isn't wrong, just confusing).
However, can I please ask that you change "found" to "find"? That way, the conjugation matches the tense, since it's an irregular verb.